components/authorization/kcp/kustomization.yaml

```diff
@@ -2,6 +2,7 @@ resources:
 - view-everything.yaml
 - members-view-everything.yaml
 - application-service-secrets.yaml
+- skabashn-spi-secrets.yaml
```

Comment:
would not use your name in the file name, since it can be extended by other members